### PR TITLE
Feature/grid wp table menu

### PR DESCRIPTION
**Pull request by ulferts**
Reuses the board list menu on the my page/grid widgets.

### TODO

* ~Move more menu related components into the menu module~
* ~Refactor the wp list menu to also use the component~
* ~Place at least the non wp specific part of the context menus into a separate module~

Although these todos are dearly required, I currently do not have the time to do it.

### CODE REVIEW

File: frontend/src/app/modules/boards/board/board-list/board-list-menu.component.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ng-container class="board-list--more-menu" *ngIf="canDelete()">
```
Comment (HDinger) [inline frontend/src/app/modules/boards/board/board-list/board-list-menu.component.html:1]: The `ngIf` is too restrictive. In cases where a user is not allowed to manage boards, but to edit versions, the menu is not shown.

**Comment by HDinger** [comment]
👍 Looks good. Please have a look at the minor remark, good to merge otherwise.

File: frontend/src/app/modules/boards/board/board-list/board-list-menu.component.html (hunk 1 above)
Comment (ulferts) [reply frontend/src/app/modules/boards/board/board-list/board-list-menu.component.html:1]: The comment is valid but as I only [copied over that part of the code](https://github.com/opf/openproject/pull/7397/files#diff-f6b9c062cae11ac21c750a25af8a4c9fL28) I am not going to change that situation. Currently it should not matter anyway, as it maps to the same permission in the backend.

Comment (HDinger) [reply frontend/src/app/modules/boards/board/board-list/board-list-menu.component.html:1]: ok 👍

**Pull request merged**